src/assets/GraphqlAsset.js

```diff
@@ -0,0 +1,21 @@
+const Asset = require('../Asset');
+const gql = require('graphql-tag');
```

Comment:
this should be a local dependency in the project. you can use the localRequire util function to require relative to the project directory. see here for an example: https://github.com/parcel-bundler/parcel/blob/master/src/assets/SASSAsset.js#L10

Reply:
Ok, will use that